poc/Makefile

```diff
@@ -10,3 +10,9 @@ test:
 	sage -python vdaf.py
 	sage -python vdaf_prio3.py
 	sage -python vdaf_poplar1.py
+
+generate_test_vectors:
+	TEST_VECTOR=TRUE sage -python idpf_poplar.py
+	TEST_VECTOR=TRUE sage -python vdaf_prio3.py
+	TEST_VECTOR=TRUE sage -python vdaf_poplar1.py
+	TEST_VECTOR=TRUE sage -python xof.py
```

Comment:
This list of commands needs to be kept in sync with the files for which we generate test vectors.

Luckily, I don't think this is necessary. To set the `TEST_VECTOR` variable to `true`, just do:
```
make TEST_VECTOR=true
```

```suggestion
```

Reply:
Good point. Done.